css/styles.css

```diff
@@ -0,0 +1,346 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+.width-100 {
+  width: 100%;
+}
+
+.flex {
+  display: flex;
+}
+
+.space-around {
+  justify-content: space-around;
+}
+
+.talign-center {
+  text-align: center;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+header {
+  background-color: rgb(218, 218, 218);
+  align-content: center;
+  height: 76px;
+  padding-top: 1%;
+  padding-bottom: 1%;
+}
+
+.margin-top1 {
+  margin-top: 1%;
+}
+
+.height-100 {
+  height: 100%;
+}
+
+.textdeco-none {
+  text-decoration: none;
+}
+
+nav > a {
+  font-size: 26px;
+  font-weight: bold;
+  padding: 10px;
+}
+
+footer {
+  background-color: #5a5a5a;
+  align-items: center;
+  padding-top: 40px;
+  padding-bottom: 40px;
+}
+
+ul {
+  list-style: none;
+  display: inline-flex;
```

Comment:
- [ ] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

Please apply DRY CSS to prevent repeating the same rules.